doc/reportv2.Rmd

```diff
@@ -218,21 +222,14 @@ ggplotly(colourplz)
 
-Third, because this dataset represents a compilation of work done by various studies, this .. For instance,
+A third pattern is also possible but involves limitations of the dataset. It is difficult to infer any general trend about the distribution and variation of predator-prey body size relationships across marine ecosystems and across feeding behaviours due to the dataset containing multiple incidences of nonindependent data. While the dataset contained over 30,000 entries, much of it represented nonindependent data, which may have substantially decreased the sample size. For instance, out of the whole dataset, 3581 samples were from the Albacore tuna. To account for nonindependence, we used a linear mixed effects model. Statistically, then, these samples represent a single unit -- instead of having 3581 samples, there is only 1 sample with 3581 values. As a result, this dataset was limited in what it could actually communicate in terms of how predator-prey body size relationships vary across marine habitats and feeding types. In fig. 4, the predacious subgroup in the shelf habitat has a negative correlation, suggesting that as prey become smaller, predators are becoming larger. Within the shelf habitat, there are 2022 samples, but only two populations are represented: the Atlantic bluefin tuna and the longfin squid. Because we accounted for nonindependence, the trend in the shelf environment is only being driven by these two populations, which statistically, amounts to two units. Evidently, a sample size of two is not enough for making any definitive conclusions about the shelf marine ecosystem, or about predator-prey body size relationships in the marine environment in general.
```

Comment:
At the same time, it could be a strength to know the distribution of values for each species, rather than just having a single (possibly extreme) observation for each species.